include/maxminddb.h

```diff
@@ -161,6 +161,7 @@ typedef struct MMDB_ipv4_start_node_s {
 } MMDB_ipv4_start_node_s;
 
 typedef struct MMDB_s {
+    bool file_mapping;
```

Comment:
True. It could be problematic for people linking dynamically. I was hesitant to add a flag because I was unsure how flags where used, if they represent undefined file flags (potentially being completely reserved) or if you can just allocate a new one.